spawn_statestores/statestores/lib/statestores/migrator/migrator.ex

```diff
@@ -5,13 +5,13 @@ defmodule Statestores.Migrator do
   def migrate(adapter) do
     load_app()
 
-    adapter.migrate()
```

Comment:
I liked this approach because it allowed a better adaptation of adapters not based on Ecto